data/json/items/gun/40.json

```
@@ -134,7 +134,7 @@
     "copy-from": "smg_base",
     "looks_like": "hk_mp5",
     "type": "GUN",
-    "name": { "str": "Luty SMG: .40 S&W", "str_pl": "Luty SMGs: .40 S&W" },
+    "name": { "str": ".40 S&W Luty SMG" },
```

```suggestion
    "name": { "str": ".40 S&W Luty submachine gun" },
```
same as `9x19mm Luty submachine gun`?